README.md

```diff
@@ -574,7 +686,7 @@ An object with the following keys:
 
 ##### Example
 
-![Bank Params Android](https://cloud.githubusercontent.com/assets/1286226/26419755/6c801f06-40c9-11e7-972e-521850eda2ef.gif)
```

Comment:
@flavordaaave I think, this is the last question why did you remove this?

Reply:
Hmm, sorry. This should not be removed. Something went wrong here. Can you just ignore that change or should I commit an updated version where this line is till in?

Reply:
I can resume this line in your fork - no problem